e2e/cypress/integration/account_settings/display/account_settings_position_spec.js

```diff
@@ -27,20 +27,18 @@ describe('Account Settings > General > Position', () => {
     });
 
     it('MM-T2063 Position', () => {
-        // # Open the hamburger menu
-        cy.findByLabelText('main menu').should('be.visible').click();
+        // # Open 'Account Settings' modal and view the default 'General Settings'
+        cy.uiOpenAccountSettingsModal().within(() => {
+            // # Open 'Position' setting
+            cy.findByRole('heading', {name: 'Position'}).should('be.visible').click();
 
-        // # Click on Account settings menu item
-        cy.findByText('Account Settings').should('be.visible').click();
+            // # Enter new 'Position'
+            cy.findByRole('textbox', {name: 'Position'}).should('be.visible').type(position);
 
-        // # Fill-in the position field
-        cy.findByText('Position').should('be.visible').click();
-        cy.get('#position').type(position);
-        cy.get('#saveSetting').click();
-        cy.get('#position').should('not.be.visible');
-
-        // # Exit the modal
-        cy.get('body').type('{esc}', {force: true});
+            // # Save and close the modal
+            cy.uiSave();
```

Comment:
I noticed, `uiSave()` has always been used with `uiClose() `method.
Can we have `uiSaveAndClose()` function ?

Reply:
I agree. Added `uiSaveAndClose`. Please take a look again.